src/OrbitQt/include/OrbitQt/orbittreeview.h

```diff
@@ -57,6 +57,16 @@ class OrbitTreeView : public QTreeView {
  public slots:  // NOLINT(readability-redundant-access-specifiers)
   void columnResized(int column, int old_size, int new_size);
 
+ protected:
+  [[nodiscard]] int sizeHintForColumn(int column) const override {
+    ORBIT_CHECK(model_ != nullptr);
+    orbit_data_views::DataView* data_view = model_->GetDataView();
+    ORBIT_CHECK(data_view != nullptr);
```

Comment:
Not having a data view is probably our equivalent of no model, so maybe we also wanna return -1 here.

Reply:
Done.